tests/python/dnnl/test_bf16_operator.py

```diff
@@ -69,26 +69,26 @@ def check_operator_accuracy(sym_fp32, sym_bf16, data_shape, num_input_data=1, bf
     arg_names = sym_fp32.list_arguments()
     aux_names = sym_fp32.list_auxiliary_states()
 
-    exe_fp32 = sym_fp32._simple_bind(ctx=mx.cpu(), **data_shape)
+    #exe_fp32 = sym_fp32._simple_bind(ctx=mx.cpu(), **data_shape)
```

Comment:
Did you actually mean to comment these lines in this file?

Reply:
No, it was a mistake from other commit, thanks for pointing it out.